playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -238,29 +238,22 @@
   any_errors_fatal: true
 
   pre_tasks:
+  - name: Load lib_openshift modules
+    include_role:
+      name: lib_openshift
```

Comment:
@mtnbikenc,
I'm a big fan of using include_role.  I disagree that it is code smell.  This is just a question of organization and when the calls to the module should run.  If you need to run something in pre_, then you can include_role in pre.  If the only place where you are running module calls is in post_tasks, then I'd probably place it in among other roles in the roles: section.